src/CircuitBreaker.ts

```diff
@@ -62,12 +68,14 @@ export class CircuitBreaker extends EventEmitter {
     }
 
     async _evaluate() {
-        this._evaluatingPromiseHook = Promise.allSettled(this._probes.map(p => p.check()));
+        this._failedProbes = false;
+        this._evaluatingPromiseHook = Promise.allSettled(this._probes.map(p => p.check().catch(() => {
+            this._failedProbes = true;
+        })));
```

Comment:
should we add a test for this ?

Reply:
done here : https://github.com/scality/breakbeat/pull/11/commits/1c03a2f08292215d1dd818d48a0b711ca81869ab#